dist/server/index.js

```diff
@@ -2,13 +2,15 @@
 
 const { createClient } = require('./sproutVideo');
 const { ELEMENT_STATE } = require('../shared');
+const isNil = require('lodash/isNil');
 
 async function beforeSave(asset, { config: { tce } }) {
   const { sproutVideoApiKey: apiKey } = tce;
   const client = createClient({ apiKey });
   const { id: videoId, playable } = asset.data.video;
   if (!videoId || !playable) return deleteTemporaryAssetProps(asset);
   await processCaption(asset, client);
+  await processPosterFrame(asset, client);
   return deleteTemporaryAssetProps(asset);
```

Comment:
It only caught my eye now, but do you think it would be better to explicitly return `asset` after calling `deleteTemporaryAssetProps` to indicate that the function mutates its argument?